tmf/org.eclipse.tracecompass.tmf.core/src/org/eclipse/tracecompass/tmf/core/dataprovider/IDataProviderDescriptor.java

```diff
@@ -47,22 +46,32 @@ public enum ProviderType {
          */
         TABLE,
         /**
-         * A provider for a tree, whose entries have XY series. The x-series is time.
+         * A provider for a tree, whose entries have XY series. The x-series is
+         * time.
          */
         TREE_TIME_XY,
         /**
-         * A provider for a Time Graph model, which has entries with a start and end
-         * time, each entry has a series of states, arrows link from one series to
-         * another
+         * A provider for a Time Graph model, which has entries with a start and
+         * end time, each entry has a series of states, arrows link from one
+         * series to another
          */
         TIME_GRAPH,
         /**
          * A provider for a data tree, which has entries (rows) and columns.
+         *
          * @since 6.1
          */
         DATA_TREE,
         /**
-         * A provider with no data. Can be used for grouping purposes and/or as data provider configurator.
+         * A provider for a MIME data provider, which can be an image, report, text, file, etc.
```

Comment:
report is not a MIME type. I would remove file as well. I think we should exactly define the ones we want to support